tutorials/Tutorial For QA Registration.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# QA REGISTRATION\n",
+    "Give a visual result for regisration part to help user check results of each step. The graph includes three dimensions(sagital slice, coronal slice and axial slice) and uses red and green to indicate the rigistered output and reference image."
```

Comment:
Doesn't it use green and pink to indicate the output, not green and red?
Also, you still need more detail on what is happening inside of your QA code. Aside from the first two sentences of this tutorial, you don't explain any part of the code or what the output represents

Reply:
I will add explanations to the input and output cells.

But I think the doctrings at the top of each code cell of functions have already explained how each function works. Do you ask for any other explanation contents for these code cells?

Reply:
Let me  explain how I would layout this tutorial:
QA Registration
[brief description of what is happening]

Setup Steps
["First we have to import all relevant functions used in this qa code. qa_reg uses ndmg functions from qa_utils and gen_utils."]
[code block]

Saving figures:
[brief description about how this takes the registered brain and reference brain, setting them to the proper dimensions and sending them to plot_overlays]
[code]

Generating the qa overlay figures
[description about how this takes 3 slices from the sagital, axial, and coronal directions, colors them, and creates a graph]
[code]

Anatomical registration
[ put what you have for "Intermediate Registration Step" here]
[add a couple sentences about what the graph means (what  is pink, what is green, what does bad data look like, are there any common errors to avoid?)]

Atlas registration
[ put what you have for "Final Registration Step" here]
[add a couple sentences about what the graph means (what  is pink, what is green, what does bad data look like, are there any common errors to avoid?)]

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/a48b23981fbece4dcac6471538b1f2b786275ef1